samcli/local/docker/container.py

```diff
@@ -314,6 +334,32 @@ def wait_for_logs(self, stdout=None, stderr=None):
 
         self._write_container_output(logs_itr, stdout=stdout, stderr=stderr)
 
+    def wait_for_socket_connection(self):
```

Comment:
Can we make this method protected since it is only used by another method in this class?

Reply:
done 👍